src/Cli.Tests/Snapshots/AddEntityTests.AddEntityWithAnExistingNameButWithDifferentCase.verified.txt

```diff
@@ -0,0 +1,102 @@
+﻿{
+  DataSource: {
+    DatabaseType: MSSQL,
+    Options: {
+      set-session-context: {
+        ValueKind: True
+      }
+    },
+    DatabaseTypeNotSupportedMessage: The provided database-type value: MSSQL is currently not supported. Please check the configuration file.
```

Comment:
why this error? MSSQL should be supported?

Reply:
It's from serialization, it can be ignored in the snapshots (I just hadn't done that at present)

Reply:
Ignored from serialization and updated all snapshots